iguana.corecontroller/src/test/java/org/aksw/iguana/cc/utils/FileUtilsTest.java

```diff
@@ -29,6 +51,22 @@ public void readLineAtTest() throws IOException {
         assertEquals("dfe", FileUtils.readLineAt(4, f));
         //read line at -1
         assertEquals("", FileUtils.readLineAt(-1, f));
```

Comment:
I think this should throw. Or is there any valid use-case for this?

Reply:
IndexedLineReader now throws an exception, when a line doesn't exist with a specified index.